### PR TITLE
Change how we handle default CSS styling

### PR DESCRIPTION
**Pull request by Norman-Amy**
This is needed because ie and edge don't support CSS variables

### CODE REVIEW

File: dialog-el.html [hunk 1]
```diff
@@ -14,11 +14,14 @@
     }
 
     .content {
-      background: var(--dialog-el-background, #FFFFFF);
-      border-radius: var(--dialog-el-border-radius, 4px);
+      background: #FFFFFF;
+      background: var(--dialog-el-background);
```
Comment (jshcrowthe) [inline dialog-el.html]: This should be changed to

```css
background: var(--dialog-el-background, #FFFFFF);
```

File: dialog-el.html [hunk 2]
```diff
@@ -14,11 +14,14 @@
...
+      border-radius: 4px;
+      border-radius: var(--dialog-el-border-radius);
```
Comment (jshcrowthe) [inline dialog-el.html]: This should be changed to

```css
border-radius: var(--dialog-el-border-radius, 4px);
```

File: dialog-el.html [hunk 3]
```diff
@@ -14,11 +14,14 @@
...
       box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.35), 0px 3px 2px 0px rgba(0,0,0,0.18);
       opacity: 0;
-      padding: var(--dialog-el-padding, 15px);
+      padding: 15px;
+      padding: var(--dialog-el-padding);
```
Comment (jshcrowthe) [inline dialog-el.html]: This should be changed to

```css
padding: var(--dialog-el-padding, 15px);
```

File: dialog-el.html [hunk 4]
```diff
@@ -74,7 +77,8 @@
       left: 50%;
       transform: rotate(45deg) translate3d(-50%,-5px,0);
       z-index: -1;
-      background-color: var(--dialog-el-background, #FFFFFF);
+      background-color: #FFFFFF;
+      background-color: var(--dialog-el-background);
```
Comment (jshcrowthe) [inline dialog-el.html]: This should be changed to

```css
border-radius: var(--dialog-el-background, #FFFFFF);
```

**Review by jshcrowthe — CHANGES_REQUESTED** [review]

File: dialog-el.html (hunk 4 above)
Comment (Norman-Amy) [reply dialog-el.html]: What is the benefit of having the fallback value there as well as in the line above it?

**Review by jshcrowthe — APPROVED** [review]

**Comment by jshcrowthe** [comment]
LGTM 👍

**Pull request merged**